sanssouci/post_hoc_bounds.py

```diff
@@ -73,10 +74,10 @@ def min_tp(p_values, thr):
 
     p_values : 1D numpy.array
         A 1D numpy array of p-values for the selected items
-    thr : 1D numpy.array
+    thresholds : 1D numpy.array
         A 1D numpy array of non-decreasing k-FWER-controlling thresholds
 
-    Returns
+    Retursubset_size
```

Comment:
oops. Normally, such a line should trigger an error. If not, it means that the test suite is not complete.